### PR TITLE
Add support for viewing estimated revenue in other currencies

### PR DESCRIPTION
**Pull request by williambl**
First thing I've ever done in React (and like the second thing I've done in JS) so please tell me if there's something I've done wrong or non-idiomatically.

I want to make the second dropdown sync with the first one, once I've worked out how to do that I'll mark this PR as ready.

### CODE REVIEW

File: src/utils/parseAnalytics.js [hunk 1]
```diff
@@ -51,8 +55,8 @@ export default function parseAnalytics(analytics) {
     cursePercentage: ((curseDownloadSum / downloadSum) * 100),
     twitchPercentage: ((twitchDownloadSum / downloadSum) * 100),
     pointDailyAverage,
-    estimatedRevenues: pointSum * PTS_USD_RATE,
-    dailyEstimatedRevenues: pointDailyAverage * PTS_USD_RATE,
+    estimatedRevenues: [ pointSum * PTS_USD_RATE, pointSum * PTS_GBP_RATE, pointSum * PTS_EUR_RATE, pointSum * PTS_CAD_RATE ],
```
Comment (antoinegag) [inline src/utils/parseAnalytics.js]: IMO if we want to implement different currencies the best would be to just not compute this and use the `pointSum` multiplied by the rate for each dynamically when we display it in the Stats

**Comment by antoinegag** [comment]
@williambl Cool idea, I can probably help you with the sync part tomorrow or this weekend, kinda busy atm but I'll find a spot for that :)

**Comment by williambl** [comment]
I've added synchronisation, it works but I don't know if I've done it properly.

File: src/utils/parseAnalytics.js (hunk 1 above)
Comment (williambl) [reply src/utils/parseAnalytics.js]: I've moved the calculation over to the component rendering in commit 6b016a9.

**Comment by williambl** [comment]
I'm marking this ready for review because it now has all the functionality I was aiming for.

**Comment by antoinegag** [comment]
Overall ok, refactored it quite a bit tho

- Moved to a single drop down to pick the currency (which makes more sense IMO)
- Query the exchange rates from a free public API so it's always up to date!

image

**Comment by williambl** [comment]
Ah nice, thanks!

**Pull request merged**